### DISCUSSION TITLE
LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.'

### DISCUSSION THREAD

**Discussion by ConstantinoSchillebeeckx** [community/community#27175] [Actions]
<p>I am running some Python unit tests on Github actions that interact with a MySQL database; the tests pass successfully when run locally, however in the action I’m seeing: <code>MySQLdb._exceptions.OperationalError: (2068, 'LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.')</code></p>
<p>The failing test creates a named temporary file (in <code>/tmp</code>) and then tries to load it locally into the MySQL DB (that I’ve manually started).</p>
<p>I have confirmed that:</p>
<ul>
<li>the DB user has FILE privileges</li>
<li>the client connects to the DB with <code>local-infile=1</code>
</li>
<li>the server has <code>local_infile</code> ON</li>
<li>
<code>secure_file_priv</code> is set to /tmp</li>
<li>the temporary file is world readable</li>
</ul>
<p>I’m afraid I don’t really know what’s happening. I’m wondering if this is an issue with the tests being run inside a container (the owner of the temporary files is <code>docker</code>)? Do I need to mount <code>/tmp</code> somehow?</p>

**Comment by ConstantinoSchillebeeckx** [comment, marked as answer]
<p>In my case, I changed the SQL call to <code>LOAD DATA INFILE</code> (note the absent <code>LOCAL</code>) - this worked for me as the file was expected to be on the server to being with.</p>

**Reply by edwardsun007** [reply]
@ConstantinoSchillebeeckx 
I am having the same issue, my db instance is running locally in docker container, and in production it is RDS aurora instance.  I want to know how you made the `tmp` directory world readable ?much appreciated
